src/index.html

```diff
@@ -5,9 +5,287 @@
+          </li>
+          <li class="nav__item">
+            <a href="#Gallery" class="nav__link">Gallery</a>
+          </li>
+          <li class="nav__item">
+            <a href="#consultation" class="nav__link">Consultation</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Testimonials" class="nav__link">Testimonials</a>
+          </li>
+          <li class="nav__item">
+            <a href="#Contacts" class="nav__link">Contacts</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section class="page__container container">
+        <div class="info" id="Info">
+          <h1 class="info__heading">
+            Miami <strong>best</strong> Real estate <a href="#" class="link">here</a>
+          </h1>
+
+          <div class="info__text">
+            <p><strong>Over 50 years experience in love.</strong><br>Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
+            <p>The market leader in real estate since 2003.<br>Awards Real Estate Company, 2010, 2011, 2013.<br>More than 200 satisfied customers premium segment.<br>We focus only on new buildings and high-level doskanalno know this area better than other specialists.<br>Working directly with developers - so have more useful information about the profitable deals.
+            </p>
+            <p><strong>We will never be late!<br>Guaranteed secure transaction - 100%.</strong></p>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__container container">
+        <h2 class="heading container__heading" id="Gallery">Gallery<span class="heading__crown"></span></h2>
+
+        <div class="gallery">
+          <div class="photos photos--wide_left">
+            <img
+              src="images/photos/landscape-1.jpg"
+              alt="North Miami Beach"
+              class="photos__img"
+            >
+            <img
+              src="images/photos/landscape-2.jpg"
+              alt="North Miami Beach"
+              class="photos__img"
+            >
+          </div>
+
+          <p class="gallery__caption">
+            Marina Palms / <a href="https://goo.gl/maps/p2SgvHMSXhr39H6m8"
+            class="link" target="blank">North Miami Beach, FL 33162</a>
+          </p>
+
+          <div class="photos">
+            <img
+              src="images/photos/landscape-3.jpg"
+              alt="Miami Beach"
+              class="photos__img"
+            >
+            <img
+              src="images/photos/landscape-4.jpg"
+              alt="Miami Beach"
+              class="photos__img"
+            >
+          </div>
+
+          <p class="gallery__caption">
+            Edition Residens / <a href="https://goo.gl/maps/jordF84Jfdju4dqD9"
+            class="link" target="blank">Miami Beach, FL 33139</a>
```

Comment:
```suggestion
            Edition Residens /
            <a
              href="https://goo.gl/maps/jordF84Jfdju4dqD9"
              class="link" 
              target="blank"
            >
              Miami Beach, FL 33139
            </a>
```

Пожалуйста учи форматирование, это важнее чем кажется